tests/features/front-end/list-view/view-layout.feature

```diff
@@ -2,33 +2,61 @@ Feature: layout and visual elements of an event in List View
 
 Background:
     Given I am a Visitor on a site with TEC
-    Given List View is enabled under Events --> Settings --> Display
-    Given the site timezone is America/Los_Angeles
-    Given "Number of events to show per page" is set to 10
+    And List View is enabled under Events --> Settings --> Display
+    And the site timezone is America/Los_Angeles
+    And "Number of events to show per page" is set to 10
 
 Scenario: display of single-day timed event in List View
+    Given I have a single-day timed event
+    Then I should see the event date as ["start date" "Date time separator" "start time" "Time range separator" "end time"]
```

Comment:
The `Then` step is flaky. There are a lot of unspoken assumptions in there and I'm not sure what you're trying to achieve here.
If you're relying on default values then hard-code those in else the test would pass even if those are empty.
`Then I should see the "event date" has the format "Y:m:d H:i:s"` is clearer.